nextcord/application_command.py

```diff
@@ -1430,8 +1460,16 @@ def __init__(
         self.description_localizations = cmd_arg.description_localizations
         self.choice_localizations = cmd_arg.choice_localizations
 
-        self.min_value = cmd_arg.min_value
-        self.max_value = cmd_arg.max_value
+        if annotation_min_value is not MISSING:
+            self.min_value = annotation_min_value
+        else:
+            self.min_value = cmd_arg.min_value
+
+        if annotation_max_value is not MISSING:
+            self.max_value = annotation_max_value
+        else:
+            self.max_value = cmd_arg.max_value
```

Comment:
`cmd_arg` should take precedence here for consistency.

If it's explicitly set in the SlashOption, the annotation should be disregarded.

```suggestion
        self.min_value = annotation_min_value if annotation_min_value is not MISSING else None
        if cmd_arg.min_value is not None:
            self.min_value = cmd_arg.min_value
        
        self.max_value = annotation_max_value if annotation_max_value is not MISSING else None
        if cmd_arg.max_value is not None:
            self.max_value = cmd_arg.max_value
```